topics/icrc_29_window_post_message_transport.md

```diff
@@ -40,10 +39,16 @@ The message `targetOrigin` should be set to the previously received `icrc29_stat
 }
 ```
 
-> **Note**: The relying party should send `icrc29_status` messages in short intervals. It is expected that some of the messages will be lost due to being sent before the signer is ready.
+### Establishment
+
+The connection is considered established once the relying party receives a `"result": "ready"` response to an `icrc29_status` message, which may not necessarily be the first message sent. In case the relying party does receive any `"result": "ready"` response within a reasonable timeframe, it should treat this as a failure to establish the connection.
+
+Once the connection is established, the relying party should continue sending `icrc29_status` messages at regular intervals to maintain the connection.
```

Comment:
"could" instead of "should"?

Polling to maintain the connection open is not a must or, is it?

Reply:
It's a must, else a dapp would need to implement multiple approaches to keep track of a connection still being open instead of the single approach described here.

I changed it from should to must to be more clear in this regard.